jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/HttpChannelState.java

```diff
@@ -866,7 +866,7 @@ protected boolean onError(Throwable th)
                 catch (Throwable x)
                 {
                     if (LOG.isDebugEnabled())
-                        LOG.debug("{} while invoking onError listener {}", x, listener, x);
+                        LOG.atDebug().setCause(x).log("{} while invoking onError listener {}", x, listener);
                     else
                         LOG.warn("{} while invoking onError listener {}", x, listener);
```

Comment:
You should not change the level you log at based on what level is enabled on the logger. The `isXxxxEnabled` methods should generally only be used for performance optimisation, avoiding building objects to pass them to a logger that will then discard them.